dataikuapi/dss/recipe.py

```diff
@@ -1313,6 +1314,60 @@ def with_input_model(self, model_id):
         return self._with_input(model_id, self.project.project_key, "model")
 
 
+class EvaluationRecipeCreator(DSSRecipeCreator):
+    """
+    Builder for the creation of a new "Evaluate" recipe, from an
+    input dataset, with an input saved model identifier
+
+    .. code-block:: python
+
+        # Create a new prediction scoring recipe outputing to a new dataset
+
+        project = client.get_project("MYPROJECT")
+        builder = EvaluationRecipeCreator("my_scoring_recipe", project)
+        builder.with_input_model(saved_model_id)
+        builder.with_input("dataset_to_evaluate")
+
+        builder.with_output("output_scored")
+        builder.with_output_metrics("output_metrics")
+        builder.with_output_evaluation_store(evaluation_store_id)
+
+        new_recipe = builder.build()
+    
+    Outputs can be created with
```

Comment:
I would be more assertive. Something like:
"Outputs must exists. You can create them with"